src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -381,6 +390,35 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('key_encryption_key', help="Key vault key uri. To enable automated rotation, provide a version-less key uri. For manual rotation, provide a versioned key uri.")
         c.argument('identity', help="client id of managed identity, resource name or id of user assigned identity. Use '[system]' to refer to the system assigned identity")
 
+    with self.argument_context('acr connected-registry') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'], help='The login server of the Cloud ACR registry. Must be the FQDN to support also Azure Stack.', required=True)
+        c.argument('connected_registry_name', options_list=['--name', '-n'], help='Name for the Connected ACR. Name must start with a letter and contain only alphanumeric characters (including ‘_’) optionally separated by a ‘-’. 5 to 40 chars long. Name must be unique under the Cloud ACR hierarchy.', required=True)
+        c.argument('resource_group_name', options_list=['--resource-group', '-g'], help='The name of the resource group where the Connected ACR registry will be created. Must be the same as the resource group for the --registry.', required=False)
+        c.argument('parent', options_list=['--parent', '-p'], help='The name of the parent Connected Registry.', required=False)
+        c.argument('repositories', options_list=['--repository', '-t'], nargs='+', help='Specifies the repositories that need to be synched to the Connected ACR. It can be in the format [REPO01] [REPO02]...', required=False)
+        c.argument('sync_token_name', options_list=['--sync-token'], help='Specifies the sync token needed to be synched to the Connected ACR. It can include access for multiple repositories.', required=False)
```

Comment:
@rosanch "Specifies the sync token used to synchronize the connected registry with its parent."